packages/blade/src/components/Typography/Title/Title.tsx

```diff
@@ -24,6 +25,7 @@ const getProps = ({
     fontStyle: 'normal',
     lineHeight: '4xl',
     fontFamily: 'text',
+    accessibilityRole: makeAccessible({ role: 'heading' }).accessibilityRole as string,
```

Comment:
we can pass `accessibilityProps` which will accept the a11y props we support and then call `makeAccessible` in the base component or the component where all the magic happens.

Reply:
@kamleshchandnani this is a lot cleaner but now in web we have role="heading" in the final html output :(

Reply:
Fixed it by checking the platform: https://github.com/razorpay/blade/pull/501/commits/85d38cc52f03e2db69600e405cd3e6d7162aa43a#diff-5bd329389c4afd8a481956fafa1cedbe640850ef03783703e887172939c4af41R58